caiman_napari/main_offline_gui.py

```diff
@@ -194,19 +192,25 @@ def show_cnmf_params_gui(self):
     def show_mcorr_params_gui(self):
         self.mcorr_gui = MCORRWidget(parent=self)
         self.mcorr_gui.show()
+
+    def temp_func(self):
+        print("show results")
+        item_gui = QtWidgets.QListWidgetItem = self.ui.listWidgetItems.currentItem()
+        uuid = item_gui.data(3)
+        ix = self.dataframe[self.dataframe['uuid'] == uuid].index[0]
+        algo = self.dataframe['algo'][ix]
+        self.viewer.open(self.dataframe['input_movie_path'][ix])
+        load_output_mcorr(self.viewer, self.dataframe.iloc[ix])
```

Comment:
Isn't it this line that does the loading of the mcorr movie? I don't see what the previous line does.

Reply:
The previous line opens the original movie, the latter opens the mcorr movie. Figured it might be better to allow for comparison, but I can remove it or we can rethink how best to compare the two (mean image, CC?)

Reply:
oh right, we should ask how to view them side by side

Reply:
Added this to the issue tracker as a "nice to have"